.github/workflows/tests-pro-integration.yml

```diff
@@ -219,10 +227,9 @@ jobs:
           path: |
             localstack/.filesystem/var/lib/localstack
           # include the matrix group (to re-use the var-libs used in the specific test group)
-          key: community-it-${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-libs-${{ hashFiles('localstack-ext/**/packages.py','localstack-ext/packages/*.py', 'localstack/**/packages.py',  'localstack/packages/*.py') }}-${{steps.determine-companion-ref.outputs.result}}-group-${{ matrix.group }}
+          key: community-it-${{ runner.os }}-python-${{ steps.setup-python.outputs.python-version }}-libs-${{ '**/packages.py', '**/packages/*') }}-${{steps.determine-companion-ref.outputs.ref}}-group-${{ matrix.group }}
```

Comment:
looks like the call to `hashFiles` went missing here?

Reply:
Good catch! Thanks, fixed!